src/App.jsx

```diff
@@ -1,5 +1,15 @@
+import Header from './Layouts/Header/Header';
+import Main from './layouts/Main/Main';
+import Footer from './layouts/Footer/Footer';
+
 function App() {
-  return <div>Hello world!</div>;
+  return (
+    <>
+      <Header />
+      <Main />
```

Comment:
are u sure that u need main component only to call weatherForecast?

Reply:
I planned to use the Main component as a wrapper for info, feedback and WeatherForecast pages